hydra-node/test/Hydra/Chain/Direct/Contract/FanOut.hs

```diff
@@ -119,6 +125,8 @@ genFanoutMutation (tx, _utxo) =
     , SomeMutation (Just $ toErrorCode LowerBoundBeforeContestationDeadline) MutateValidityBeforeDeadline . ChangeValidityInterval <$> do
         lb <- genSlotBefore $ slotNoFromUTCTime healthyContestationDeadline
         pure (TxValidityLowerBound lb, TxValidityNoUpperBound)
+    , SomeMutation (Just $ toErrorCode BurntTokenNumberMismatch) MutateThreadTokenQuantity
+        <$> changeMintedTokens tx (valueFromList [(AssetId (headPolicyId testSeedInput) hydraHeadV1AssetName, 1)])
```

Comment:
At first I thought this is not doing what is claimed on line 108. It's a bit tricky to see that based on the `Monoid` instance of `Value` and the `AssetId` here being essential the ST token of our protocol, this will change the minted value to _not burn the ST_.

Can we maybe make this easier to read and more quickchecky by just selecting any of the burned assets and just dropping the one selected?